fvm/evm/emulator/emulator.go

```diff
@@ -321,7 +325,7 @@ func (bl *BlockView) newProcedure() (*procedure, error) {
 		config: cfg,
 		evm: gethVM.NewEVM(
 			*cfg.BlockContext,
-			*cfg.TxContext,
+			// *cfg.TxContext, TODO(m-Peter): Investigate
```

Comment:
Any information on this TODO?

Reply:
Good call out :+1: After some searching, I found out that you can manually set it now with `SetTxContext`. Updated in: https://github.com/onflow/flow-go/pull/7241/commits/0fbe41d7d9736110f4e12fe6873fc4608fa5fe75 .